src/libs/blueprint/conduit_blueprint_mesh.cpp

```diff
@@ -1084,12 +1084,12 @@ convert_oneD_coordset_to_strip(const conduit::Node &coordset,
     if (coord_type == "uniform")
     {
         copy_node_rename_first_child(coordset, dest, "dims", "j", "i");
-        dest["dims/i"].set(1);
+        dest["dims/i"] = 1;
```

Comment:
I am still confused about this, is:

```
copy_node_rename_first_child(coordset, dest, "dims", "j", "i");
dest["dims/i"] = 1;
```

equiv to:

```
dest["dims/j"] = coordset["dims/i"];
dest["dims/i"] = 1;
```

?

If so, the helper is obfuscating this, I suggest we just do the direct operations. 

Also, for the result coords for the strip mesh, is there a reason we want the first logical coord to be flat vs instead of the second?

Why not use:
```
dest["dims/j"] = coordset["dims/i"];
dest["dims/j"] = 1;
```

It seems strange to swap the coords instead of just extending?

Reply:
The helper is probably due to my inexperience.  I was trying to do:

```
dest["dims/j"] = coordset["dims/i"];
dest["dims/i"] = 1;   // being sure that dest["dims/i"] has the same data type as dest["dims/j"]
```

If the extra step isn't needed, I'll happily do it that way.

There's a library that expects the strip mesh to have y as the independent variable, and x to be [0, 1].

Reply:
Removed the helper function; using the pattern you described above.